src/router/index.js

```diff
@@ -0,0 +1,70 @@
+import Vue from "vue";
+import VueRouter from "vue-router";
+import NotFound from "@/views/NotFound";
+
+Vue.use(VueRouter);
+
+const routes = [
+  {
+    path: "",
+    component: () => import("../views/Dashboard"),
+    children: [
+      {
+        path: "/",
+        name: "home",
+        component: () => import("../views/Home"),
+      },
+      {
+        path: "/tags",
+        name: "tags.list",
+        component: () =>
+          import(/* webpackChunkName: "additional" */ "../views/Tags"),
+      },
+      {
+        path: "/users",
+        name: "users.list",
+        component: () =>
+          import(/* webpackChunkName: "additional" */ "../views/users/List"),
+      },
+      {
+        path: "/users/:id(\\d+)",
+        name: "user.profile",
+        props: true,
+        component: () =>
+          import(/* webpackChunkName: "additional" */ "../views/users/View"),
+      },
+      {
+        path: "/questions/",
+        name: "questions.list",
+        component: () =>
+          import(
+            /* webpackChunkName: "additional" */ "../views/questions/List"
+          ),
+      },
+      {
+        path: "/questions/:id(\\d+)",
+        name: "answered.question",
```

Comment:
the name is not descriptive. Why `question` is second word, why `answered`? When you post the question it's not answered, but the link will be the sames.
Better ti name it: `questions.view` - as the name of folder and component

Reply:
Fixed